plugins/wporg-5ftf/includes/email.php

```diff
@@ -205,3 +206,52 @@ function send_pledge_deactivation_email( $pledge ) {
 		$pledge->ID
 	);
 }
+
+/**
+ * Ask an inactive contributor to update their pledge for accuracy.
+ */
+function send_contributor_inactive_email( array $contributor ) : bool {
+	/*
+	 * Their first name is ideal, but their username is the best fallback because `nickname`, `display_name`,
+	 * etc are too formal.
+	 */
+	$name    = $contributor['first_name'] ? $contributor['first_name'] : '@' . $contributor['user_nicename'];
+	$subject = 'Please update your Five for the Future pledge';
+
+	$short_team_names = array_map( function( $team ) {
+		$team = str_replace( 'Team', '', $team );
+
+		return trim( $team );
+	}, $contributor['teams_names'] );
+
+	$message = sprintf( "
```

Comment:
Should there be a last-chance check to make sure all the variables are set? Just to guard against silly errors like "you pledged 0 hours" and other things that are likely to wind up in a screenshot on social media :)

Reply:
🤔 

* `$name` falls back to `user_nicename`, which should be guaranteed to exist
* `$contributor['hours_per_week']` and `team_names` have checks to make sure they're valid

https://github.com/WordPress/five-for-the-future/blob/8d6ea36d2fd998b4b18bd178198e83cd3f0141c5/plugins/wporg-5ftf/includes/contributor.php#L650-L651

https://github.com/WordPress/five-for-the-future/blob/8d6ea36d2fd998b4b18bd178198e83cd3f0141c5/plugins/wporg-5ftf/includes/contributor.php#L666-L669

It feels a bit WET/overkill to duplicate that in the email function, but I can see the concern.